Automattic/jetpack
# E2E: Add wordads block test

#### Changes proposed in this Pull Request:

* Adds wordAds block E2E test

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* p9dueE-Ju-p2

#### Testing instructions:






* Check Travis build
* Run tests manually. You need a clean disconnected site without any plan or a connected site with a sandboxed Pro plan. newly created JN site works perfectly. Or you can create a dockerized disposable site using this PR: https://github.com/Automattic/jetpack/pull/12842

With JN site:
- create a new site with installed Jetpack
- Get the JN url and password, which you'll need to pass it to tests
- run tests `PUPPETEER_HEADLESS=false WP_BASE_URL=${WP_SITE_URL} WP_PASSWORD=${JN_PASSWORD} WP_USERNAME="demo" yarn test-e2e --runInBand`

#### Proposed changelog entry for your changes:

* N/A
